app/views/includes/countries.scala.html

```diff
@@ -14,15 +14,25 @@
  * limitations under the License.
  *@
 
-@import utils.CountryCodes
+@import _root_.utils.CountryCodes
 
-@(countryVal: Option[String])(implicit countryCodes: CountryCodes)
+@this(govukSelect: GovukSelect)
 
-<label for="country"></label><select id="country"  class="drop-down" name="country">
-    <option value="">Select...</option>
-    @for(country <- countryCodes.countriesMap.sortBy(_.country)) {
-    <option value = @country.countryCode @if(countryVal.contains(country.countryCode)) {selected}>
-    @country.country
-    </option>
-    }
-</select>
+@(countryVal: Option[String])(implicit countryCodes: CountryCodes, messages: Messages)
+
+@govukSelect(Select(
+    id = "country",
+    name = "country",
+    classes = "govuk-!-width-one-half",
+    items = Seq(SelectItem(value = Some(""), text = "Select...")) ++
```

Comment:
in the future this (`Select...`) will probably need to be changed to https://github.com/alphagov/govuk-country-and-territory-autocomplete, but for now could you include this among the Welsh-needed texts?